oonf-olsrd2/Config.in

```diff
@@ -32,4 +32,16 @@
 			The remotecontrol plugin allows you to control configuration and logging over the telnet plugin. Be careful not to open this functionality over the network without securing it.
 		default n
 
+	config OONF_GENERIC_HTTP
+		bool "http plugin enabled"
```

Comment:
Shouldn't this be "HTTP plugin enabled"? Just a bit of nitpicking while the code is compiling. :-)

Reply:
Since it's already merged, maybe it can be for consideration on the next update.

Reply:
Not sure it is necessary... typically the HTTP Plugin just acts as a gateway to the telnet plugin. You can access everything just by opening a TCP connection to the telnet port of OONFm e.g.:

echo /nhdpinfo link | nc 127.0.0.1 2009

Reply:
I was only talking about writing http upper case.

Reply:
Ahh *G*
sure, will commit this change today to the OONF repository... Thinking about releasing 0.15.0 soon anyways, got a few mayor upgrades since the last tagged version.